src/root.tsx

```diff
@@ -0,0 +1,88 @@
+import { useIntrospectionStore } from "./state/introspection";
+import { invariant } from "./utils/invariant";
+import CancelIcon from "@mui/icons-material/Cancel";
+import {
+  Box,
+  FormControl,
+  IconButton,
+  InputLabel,
+  MenuItem,
+  Select,
+  SelectChangeEvent,
+} from "@mui/material";
+import { useMemo } from "react";
+import { Outlet } from "react-router-dom";
+import { theme } from "./lib";
+import { useAppConfigQuery } from "./hooks/use-appconfig-query";
+
+function Root() {
```

Comment:
Is "Root" a good name here ? 
I dont know what you meant with it until i checked this branch out and tested it. 
Would sth. like "DebugPanel/FunctionPanel/..." describe this component better ?

Reply:
It's actually not just the debug panel. A name like this would be confusing. It's wrapping our primary router outlet.

Other projects have a similar naming strategy, e.g. [Mattermost](https://github.com/mattermost/mattermost-webapp/blob/master/root.tsx)